mu-plugins/central-hub/bootstrap.php

```diff
@@ -81,4 +84,4 @@ function load_configurations() {
 	}
 }
 
-autoload();
```

Comment:
Added empty line at the end of the PHP file, as this will be required for WPCS, i.e. once we add the code validators/sniffers.